packages/hooks/src/computed.rs

```diff
@@ -74,14 +73,13 @@ impl<I> Tracked<I> {
     pub fn compute<O: PartialEq + 'static>(
```

Comment:
For places where a selector is used in multiple places you could previously create the selector only once, now you will need to create it in each child component and the callback will need to be rerun for each component when the tracked state changes instead of just once.

This API works, but it doesn't make much sense for compute to be public in this version. I think most users shouldn't need to know what a scope id is and now that there is no public API for Selector, I don't think anyone could get useful information out of this

I think we should depreciate selectors in favor of signals. The [use_selector](https://github.com/DioxusLabs/dioxus/blob/master/packages/signals/examples/selector.rs#L12) hook in the signals crate is much more powerful because you can directly read the signal's value and create selectors that read from multiple signals.

Reply:
_now you will need to create it in each child component_ oh, that might make sense

Sorry for taking your time!